Snackr/Views/Account/Claims.cshtml

```diff
@@ -0,0 +1,37 @@
+﻿@model Snackr.Models.UserProfileViewModel
+
+@{
+    ViewBag.Title = "Claims";
+    Layout = "_Layout";
+}
+
+<div class="row">
```

Comment:
What are you going to use this page for? If nothing I would remove it.

Reply:
it was used for debugging. to pull the email, name, and etc out of the Auth0 'claims list' i just output it to a page so i could view what it was in the claimbag. i will remove it in the future when im completely satisfied with the way login works